tests/utils.go

```diff
@@ -4579,6 +4579,9 @@ func SkipTestIfNoCPUManager() {
 		}
 	}
 	Skip("no node with CPUManager detected")
+	if !HasFeature(virtconfig.CPUManager) {
```

Comment:
Shouldn't it be moved to the beginning of the method? In this stage the test is already skipped.

Reply:
Definitely. :+1: